exchanges/kucoin/kucoin_test.go

```diff
@@ -145,6 +148,8 @@ func TestGetOrderbook(t *testing.T) {
 	t.Parallel()
 	if !areTestAPIKeysSet() {
 		t.Skip(credentialsNotSet)
+	} else if ku.CurrencyPairs.IsAssetEnabled(asset.Spot) != nil && ku.CurrencyPairs.IsAssetEnabled(asset.Margin) != nil {
```

Comment:
I would revert this change across the test file, as we should in all cases in test have spot and margin enabled. Force enable of all supported assets.